src/test/java/com/spotify/docker/client/DefaultDockerClientUnitTest.java

```diff
@@ -301,4 +306,103 @@ public void testBuildPassesMultipleRegistryConfigs() throws Exception {
     final JsonNode headerJsonNode = toJson(BaseEncoding.base64().decode(registryConfigHeader));
     assertThat(headerJsonNode, is(toJson(registryConfigs.configs())));
   }
+
+  @Test
+  public void testInspectNode() throws Exception {
+    final DefaultDockerClient dockerClient = new DefaultDockerClient(builder);
+
+    // build() calls /version to check what format of header to send
+    server.enqueue(new MockResponse()
```

Comment:
Since this is repeated many times, maybe you can abstract into separate method like `enqueueServerApiVersion(final String apiVersion)`?